my-dsl.md

```diff
@@ -2,37 +2,79 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+AWK, http://www.cs.princeton.edu/~bwk/btl.mirror/
 
 # Domain
 _Describe the language's domain in five words._
 
+Text processing and data extraction
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+AWK runs on pattern-action pairs, written as `pattern { action }`. The pattern 
+is typically an expression, and the action is a series of commands. As an AWK 
+script runs, it tries to match each pattern with the data or text it is run 
+with. If there are any matches, AWK will run the action that is paired with the 
+pattern.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+AWK was originally designed to process text and manipulate data. Thus, it is 
+mostly used in one-liner scripts/programs that do these domain-specific things. 
+However, since AWK is Turing-complete, it can be used like a general-
```

Comment:
While Turing completeness can be an attractive way to classify DSLs, don't get sucked in. It is, at best, a first-order heuristic.